src/transformers/modelcard.py

```diff
@@ -591,6 +591,8 @@ def from_trainer(
 
         if model_name is None:
             model_name = Path(trainer.args.output_dir).name
+        if not model_name:
```

Comment:
```suggestion
        if len(model_name) == 0:
```
Let's not rely on Python magic bool conversion.

Reply:
For sure! Resolved in https://github.com/huggingface/transformers/pull/20496/commits/6f880107daab2c0ffd64097f95953a9eb1be3336